src/rf/rf/settings/base.py

```diff
@@ -55,6 +55,11 @@ def get_env_setting(setting):
 # END FILE STORAGE CONFIGURATION
 
 
+# STACK COLOR CONFIGURATION
+STACK_COLOR = environ.get('RF_STACK_COLOR', 'Black')
```

Comment:
`Black` is just a default color that'll never get used in an AWS deployment (only `Blue` and `Green`).

In the context of the application, I think we may need to include color in the queue and bucket names to prevent different versions of the code from interleaving. For example, `Blue` web application version accepts a job, but a `Green` stack is coming up or is coming down, but grabs it to start processing.